sql/mob_droplist.sql

```diff
@@ -32387,7 +32387,7 @@ INSERT INTO `mob_droplist` VALUES (4440,0,0,1000,1039,70);
 INSERT INTO `mob_droplist` VALUES (4441,0,0,1000,2159,420);
 INSERT INTO `mob_droplist` VALUES (4441,0,0,1000,2165,30);
 INSERT INTO `mob_droplist` VALUES (4441,0,0,1000,2220,220);
-INSERT INTO `mob_droplist` VALUES (4442,0,0,1000,17709,800);
+INSERT INTO `mob_droplist` VALUES (4442,0,0,1000,17709,583); (Cwn Cyrff) Swan Bilbo
```

Comment:
Importing mob_droplist.sql into the database...ERROR 1064 (42000) at line 32390: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'Cwn Cyrff) Swan Bilbo
INSERT INTO `mob_droplist` VALUES (4443,0,0,1000,858,290)' at line 1